hydrolib/core/io/dflowfm/ini/models.py

```diff
@@ -129,6 +130,8 @@ def _convert_value(cls, key: str, v: Any) -> str:
             return str(int(v))
         elif isinstance(v, list):
             return cls.get_list_field_delimiter(key).join([str(x) for x in v])
+        elif isinstance(v, Enum):
```

Comment:
Nice! So this means that the `.value` is no longer necessary?

Reply:
Indeed! I did not yet do a search through all the code to remove all occurances of the .value though.